GVFS/GVFS.Service/IRepoMountProcess.cs

```diff
@@ -0,0 +1,11 @@
+﻿using GVFS.Common.Tracing;
+
+namespace GVFS.Service
+{
+    public interface IRepoMounter

```

Comment:
Do we mean to have two different `IRepoMounter` interfaces (one in `GVFS/GVFS.Service/IRepoMounter`, and another in `GVFS/GVFS.Service/IRepoMountProcess`)?

Reply:
`IRepoMountProcess` is not needed anymore. Removed it.